src/composer.json

```diff
@@ -20,7 +20,8 @@
         "laravel/tinker": "^2.5",
         "league/flysystem-aws-s3-v3": "~1.0",
         "spatie/simple-excel": "^1.13",
-        "tightenco/ziggy": "^1.0"
+        "tightenco/ziggy": "^1.0",
+        "nesbot/carbon": "^2.16"
```

Comment:
I believe [Carbon is already included as a dependency of Laravel](https://www.digitalocean.com/community/tutorials/easier-datetime-in-laravel-and-php-with-carbon#:~:text=Luckily%20for%20us%2C%20Carbon%20is,and%20add%20it%20with%20Composer.). This addition doesn't hurt, but it's not needed. Your `use Carbon\Carbon;` statement in code is still needed, but not this line.